### ISSUE TITLE
Associated CSS for columns not generated in header or footer

### ISSUE THREAD

**Issue by eichenlauba** [PrestaShop/mjml-theme-converter#28]
Hi! 
I'm testing out this converter for few days, and i realized when i've section with multiple columns like this: 
`<mj-section background-color="#ffffff" padding="32px 78px" text-align="center"  border-top="1px solid rgba(0, 0, 0, 0.08)">
  <mj-column>
    <mj-image padding="0" width="40px" height="40px" align="center" src="/assets/icon.png"></mj-image>
    <mj-text font-size="12px" line-height="16px" padding="16px 0 0 0" align="center">Paiement <br> sécurisé</mj-text>
  </mj-column>
  <mj-column>
    <mj-image padding="0" width="40px" height="40px" align="center" src="/assets/icon.png"></mj-image>
    <mj-text font-size="12px" line-height="16px"  padding="16px 0 0 0" align="center">Livraison <br> offerte</mj-text>
  </mj-column>
  <mj-column>
    <mj-image padding="0" width="40px" height="40px" align="center" src="/assets/icon.png"></mj-image>
    <mj-text font-size="12px" line-height="16px"  padding="16px 0 0 0" align="center">Retour gratuit <br> sous 14 jours</mj-text>
  </mj-column>
  <mj-column>
    <mj-image padding="0" width="40px" height="40px" align="center" src="/assets/icon.png"></mj-image>
    <mj-text font-size="12px" line-height="16px"  padding="16px 0 0 0" align="center">Service <br> après-vente</mj-text>
  </mj-column>
  <mj-column>
    <mj-image padding="0" width="40px" height="40px" align="center" src="/assets/icon.png"></mj-image>
    <mj-text font-size="12px" line-height="16px"  padding="16px 0 0 0" align="center">Garanties</mj-text>
  </mj-column>
</mj-section>`

In this case, when i generate the html.twig files, my header and my footer are generated right but not the associated css.

In my html i've class like this:
`mj-column-per-20`
But no CSS with this class ...

In conclusion all the CSS seems to be generated from layout.mjml.twig (if i create a section with 5 empty columns, the CSS is rightly generated) and ignored in footer and header.mjml.twig..

Do you have any idea how to fix this behavior?

Thanks for your help!
